packages/application-extension/style/base.css

```
@@ -33,3 +33,8 @@
 .jp-MainAreaWidget > .jp-Toolbar-micro {
   display: none;
 }
+
+#jp-NotebookLogo {
+  /* bring logo to the front so it is selectable by tab*/
+  z-index: 10 ;
```

```suggestion
  z-index: 10;
```

i think is failing cause of the space before the semi colon. https://stylelint.io/user-guide/rules/at-rule-semicolon-space-before/

---

Yes, I think it's the case. Author / @Cheukting can also run `jlpm run prettier` locally and commit changes.